powersimdata/design/investment/investment_costs.py

```diff
@@ -4,6 +4,8 @@
 import numpy as np
 import pandas as pd
 
+from powersimdata.design.compare import generation as compare_generation
+from powersimdata.design.compare import transmission as compare_transmission
```

Comment:
Why don't we do `from powersimdata.design.compare.generation import calculate_plant_difference` directly here?

Reply:
Originally I had called the generation function `calculate_difference`, but you are right that now that it has a more descriptive name, we can import the function directly rather than importing the module.

Reply:
Done.